Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -188,6 +203,26 @@ private void InitializeInternal()
             dictationEventData = new DictationEventData(EventSystem.current);
 
 #endif // UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
+
+            if (MixedRealityManager.Instance.ActiveProfile.IsSpeechCommandsEnabled)
+            {
+                SpeechInputSource = new SpeechInputSource(
+                    MixedRealityManager.Instance.ActiveProfile.SpeechCommandsProfile.SpeechCommands
+#if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
+                    , (UnityEngine.Windows.Speech.ConfidenceLevel)MixedRealityManager.Instance.ActiveProfile.SpeechRecognitionConfidenceLevel
+#endif
+                );
+            }
+
+            if (MixedRealityManager.Instance.ActiveProfile.IsDictationEnabled)
+            {
+                DictationInputSource = new DictationInputSource();
+            }
+
+            if (MixedRealityManager.Instance.ActiveProfile.IsTouchScreenInputEnabled)
+            {
+                TouchscreenInputSource = new TouchscreenInputSource();
```

Comment:
@cre8ivepark This area would be where we create our gesture input source, but I've removed it for now.